src/lib/messaging/ui/interface.es

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
This class is implemented here:
https://github.com/soyundev/syn-ui/blob/86183e7b09f194c1e7a66de97b17da0762a51d43/src/lib/messages/index.es